admin/src/app/Components/images/images.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component } from '@angular/core';
+import { HttpEventType } from '@angular/common/http';
+import { ActivatedRoute } from '@angular/router';
+
+import { environment } from 'src/environments/environment';
+import { ImageService } from 'src/app/image.service';
+import { AccountService } from 'src/app/account.service';
+import { Image } from 'src/app/Dtos/image';
+
+@Component({
+  selector: 'app-images',
+  templateUrl: './images.component.html',
+  styleUrls: ['./images.component.css']
+})
+export class ImagesComponent{
+  public progress: number = 0;
+  public images: Image[] = [];
+  private hotelId: number = 0;
+  private readonly apiUrl: string;
+
+  public get isAdmin(): boolean {
+    return this.accountService.isAdmin();
+  }
+  
+  public constructor(
+    private readonly route: ActivatedRoute,
+    private readonly imageService: ImageService,
+    private readonly accountService: AccountService
+  ) {
+    this.apiUrl = environment.api;
+    this.route.params
+      .subscribe(params => this.hotelId = params.id);
+    this.fetchImages();
+  }
+
+  public createImgPath = (image: Image): string => {
+    return `${this.apiUrl}api/hotels/${this.hotelId}/images/${image.id}`;
+  };
```

Comment:
It should be located in service because I gonna use this method in some other places in code. Have fixed it

About arrow function, mans from from internet used this function type and I didn't pay attention. Fixed this moment also.
I do not mean that using arrow function is a mistake but code seems to be more readable with usual function and there are no benefits if I use arrow function here

Reply:
Actually, there is a huge difference between arrow function and "usual function". I would like to hear it from you :)

Reply:
Usual function for me is not arrow function, not iif and not function expression. "Usual" function is not technical term, I should better not use it